netbout-client/pom.xml

```diff
@@ -180,7 +180,13 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                         <configuration>
                             <excludes>
-                                <exclude>findbugs:.*</exclude>
+                                <!--
+                                @todo #911:30min Fix findbugs errors in client
+                                 Some issues may not require fixing an should be
+                                 excluded. Filters in qulice 0.12 are broken, right
```

Comment:
@exper0 did you create an issue on Qulice project for that?
put link to the issue if yes